internal/connector/pingfederate/pingfederate_connector_test.go

```diff
@@ -56,6 +56,42 @@ func TestPingFederateTerraformPlan(t *testing.T) {
 			resource:      resources.AuthenticationPolicyContract(PingFederateClientInfo),
 			ignoredErrors: nil,
 		},
+		{
+			name:     "PingFederateAuthenticationSelector",
+			resource: resources.AuthenticationSelector(PingFederateClientInfo),
+			ignoredErrors: []string{
+				"Error: Plugin did not respond",
```

Comment:
What are all these ignored errors?

Reply:
See the google doc linked in the JIRA ticket.